meson.build

```diff
@@ -115,6 +115,7 @@ log_manager_ext_args = []
 
 subdir('extensions')
 subdir('phosphor-rsyslog-config')
+subdir('phosphor-auditlog')
```

Comment:
When this goes upstream, I think we'll want to wrap this in a meson option to only build it if auditlogging has been enabled.

Reply:
Good point. I've added this to my list of todos for the upstream work.

Reply:
The compile time meson option was added into the 1110 rebase PR: https://github.com/ibm-openbmc/phosphor-logging/pull/57